source/auth/auth.rst

```diff
@@ -610,16 +610,10 @@ follows:
 
 This same conversation over mongodb's sasl implementation would appear as follows:
 
-| C: :javascript:`{saslStart: 1, mechanism: "SCRAM-SHA-1", payload: BinData(0, "biwsbj11c2VyLHI9ZnlrbytkMmxiYkZnT05Sdjlxa3hkYXdM")}`
+| C: :javascript:`{saslStart: 1, mechanism: "SCRAM-SHA-1", payload: BinData(0, "biwsbj11c2VyLHI9ZnlrbytkMmxiYkZnT05Sdjlxa3hkYXdM"), options: { skipEmptyExchange: true }}`
```

Comment:
Reading this again, I think that it's too easy to miss this new options thing, as it's in a section that's marked "as an example".  Let's add a new paragraph right above this Conversation section that says something like:

     Drivers MUST add a top-level options field to the saslStart command, whose value is a document containing a field named skipEmptyExchange whose value is true.  Older servers will ignore the options and continue with the longer exchange shown in the [Backwards Compatibility] section.  Newer servers will ... 

and so on.   A bit awkwardly phrased but maybe you can fix that.

I'm not sure whether to copy that paragraph to SCRAM-SHA-256.  It looks like some of the text from above is duplicated for SCRAM-SHA-256 and some is not.

Reply:
I have added the paragraph to both the SCRAM-SHA-1 and SCRAM-SHA-256 sections.